cli/install.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/awslabs/aws-sdk-go/aws"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/installer"
+)
+
+func init() {
+	validInstanceTypes := strings.Join(installer.ValidEC2InstanceTypes, ", ")
+	defaultInstanceType := installer.ValidEC2InstanceTypes[0]
+
+	register("install", runInstaller, fmt.Sprintf(`
+usage: flynn install <target> [-n <instances>] [-t <instance-type>] [--aws-access-key-id <key-id>] [--aws-secret-access-key <secret>] [--aws-region <region>]
+
+Targets:
+	aws  creates a flynn cluster on EC2
+
+Options:
+  -n, --instances <instances> Number of instances to launch [default: 1]
+  -t, --type <instance-type> Type of instances to launch (%s) [default: %s]
+      --aws-access-key-id <key-id>  AWS access key ID [default: AWS_ACCESS_KEY_ID]
+      --aws-secret-access-key <secret> AWS access key secret [default: AWS_SECRET_ACCESS_KEY]
```

Comment:
I think these default statements will override the env vars, which is not what we want. Better to do it in prose: `Defaults to $AWS_ACCESS_KEY_ID`